app/src/organisms/Devices/ProtocolRun/utils/getLabwareDefinitionUri.ts

```diff
@@ -7,13 +7,14 @@ export function getLabwareDefinitionUri(
   labware: ProtocolFile<{}>['labware'],
   labwareDefinitions: ProtocolFile<{}>['labwareDefinitions']
 ): string {
-  const labwareDefinitionId = labware[labwareId].definitionId
-  if (labwareDefinitionId == null) {
+  //  @ts-expect-error
+  const labwareDefinitionUri = `${labware[labwareId].definitionUri}_id`
```

Comment:
Not sure if this is correct or if it should be:

```suggestion
  const labwareDefinitionUri = labware[labwareId].definitionUri
```

Adding `_id` at the end seems to work fine in the app but I can't figure out how to fix a few of the broken tests such as `createSnippet` because none of the v6 fixtures add `_id` in the labware definition